@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -666,6 +666,227 @@ export class TextElementsRenderer {
         });
     }
 
+    renderTextElements(
```

Comment:
This method was just moved up b/c it is now public and we have a tslint rule that public methods have to come first.